src/resources/fundingLogos/de_credit_white.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="135" height="32" viewBox="0 0 135 32" preserveAspectRatio="xMinYMin meet" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not a review comment, just a heads up; at some point I'd like to turn these SVGs into jsx or something so we can programatically generate them and save shipping a bunch of redundant SVGs. Maybe that jsx-rendering function can take the locale as a parameter, and we can push the logic down into there.

Reply:
Actually since it potentially makes this PR a whole bunch simpler, let me see how easy that would be...